src/aliceVision/multiview/resection/EPnPSolver.hpp

```diff
@@ -54,7 +54,7 @@ class EPnPSolver : public robustEstimation::ISolver<robustEstimation::Mat34Model
    */
   void solve(const Mat& x2d, const Mat& x3d, std::vector<robustEstimation::Mat34Model>& models, const std::vector<double>& weights) const override
   {
-     throw std::logic_error("EPnPSolver does not support problem solving with weights.");
+     solve(x2d, x3d, models);
```

Comment:
that is not implemented so it's wrong to do that.
For what it matters also the DLT method is implemented without the weights. It would be a good idea to add it to `solveProblem`